commands/namespaces/remove.go

```diff
@@ -20,10 +20,11 @@ const forceUninstallHint = "HINT: use --force to remove the namespace and all it
 // NewRemoveCommand returns a new command to remove an existing namespace.
 func NewRemoveCommand(l *zap.SugaredLogger) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "remove",
-		Long:    "Remove an existing namespace",
-		Short:   "Remove an existing namespace",
-		Example: `everestctl namespaces remove [NAMESPACE] [FLAGS]`,
+		Use:     "remove [flags] namespaces",
```

Comment:
Maybe we should capitalize the namespaces to visually indicate that this is something that the users should replace.
```suggestion
		Use:     "remove [flags] NAMESPACES",
```